examples/step-87/doc/intro.dox

```diff
@@ -0,0 +1,268 @@
+the interface $\Gamma$ is explicitly represented by a surface mesh. The latter is immersed into a background mesh,
+from which the local velocity at the support points of the surface mesh is extracted and leads to a movement
+of the support points of the immersed mesh as
+@f[
+	\boldsymbol{x}_q^{(i + 1)} = \boldsymbol{x}_q^{(i)} +  \Delta t \cdot \boldsymbol{u}(\boldsymbol{x}_q^{(i)}) \quad \text{ for } \boldsymbol{x}_q \in \Gamma
+@f]
+considering an explicit Euler time integration scheme  from time step $i$ to $i+1$ with time step-size $\Delta t$.
+
+For a two-phase-flow model considering the incompressible Navier-Stokes equations, the two phases are usually
+coupled by a singular surface-tension force $\boldsymbol{F}_S$, which results, together with the difference in fluid properties,
+in discontinuities across the interface:
+@f[
+	\boldsymbol{F}_S(\boldsymbol{x})= \sigma \kappa(\boldsymbol{x}) \boldsymbol{n}(\boldsymbol{x}) \delta_{\Gamma}(\boldsymbol{x}).
+@f]
+Here $\sigma$ represents the surface-tension coefficient,
+$\boldsymbol{n}(\boldsymbol{x})$ the interface normal vector
+and $\kappa(\boldsymbol{x})$ the interface mean curvature field.
+The singularity at the interface is imposed by the Dirac delta function
+@f[
+\delta_{\Gamma}(\boldsymbol{x}) = \begin{cases}
+1 & \text{on } \Gamma \\
+0 & \text{else}\end{cases}
+@f]
+with support on the interface $\Gamma$.
+In a finite element context, the weak form of the surface-tension force
+is needed. The latter can be applied as a  sharp surface-tension force model
+@f[
+	(\boldsymbol v, \boldsymbol F_S)_{\Omega} = \left( \boldsymbol{v}, \sigma \kappa \boldsymbol{n} \right)_\Gamma,
+@f]
+exploiting the property of the Dirac delta function for any smooth
+function $v$, i.e., $\int_\Omega\delta_{\Gamma}\,v\,\text{d}x=\int_\Gamma v\,\text{d}y$. For
+front-tracking methods, the curvature and the normal vector are directly computed
+from the surface mesh.
+
+Alternatively, in regularized surface-tension-force models
+@cite brackbill1992continuum @cite olsson2005conservative @cite kronbichler2018fast,
+the Dirac delta function is approximated by a smooth ansatz
+@f[
+(\boldsymbol v, \boldsymbol F_S)_{\Omega} \approx \left(\boldsymbol v, \sigma \kappa \boldsymbol{n} \|\nabla H\|  \right)_\Omega
+@f]
+considering the absolute value of the gradient of a regularized indicator function $\|\nabla H\|$,
+which is related to the
+level-set field. In such models, the interface normal vector
+@f[
+	\boldsymbol{n}(\boldsymbol{x}) = \nabla \phi(\boldsymbol{x}),
+@f]
+and the interface curvature field
+@f[
+	\kappa(\boldsymbol{x}) = \nabla \cdot \boldsymbol{n}(\boldsymbol{x})=\Delta \phi(\boldsymbol{x}) \,.
+@f]
+are derived from the level-set function.
+
+
+<h3>Overview</h3>
+
+In the following, we present three simple use cases of Utilities::MPI::RemotePointEvaluation
+(mini examples). We start with discussing a serial code in mini example 0.
```

Comment:
Why number examples from 0? The C++ programmer in me loves this, but it is a bit peculiar (we don't have a step-0 for instance)

Reply:
Sounds reasonable. If you feel it's more appropriate this way, I don't have a strong opinion on this.